### PR TITLE
add new field to FileBase

### PR DESCRIPTION
**Pull request by andezdev**
All types that extends from FileBase ([File](https://core.telegram.org/bots/api#file), [PhotoSize](https://core.telegram.org/bots/api#photosize), [Audio](https://core.telegram.org/bots/api#audio), [Document](https://core.telegram.org/bots/api#document), [Video](https://core.telegram.org/bots/api#video), [Animation](https://core.telegram.org/bots/api#animation) ...) accept the field file_unique_id

### CODE REVIEW

File: types/node-telegram-bot-api/index.d.ts [hunk 1]
```diff
@@ -161,7 +161,6 @@ declare namespace TelegramBot {
         reply_to_message_id?: number | undefined;
         reply_markup?: InlineKeyboardMarkup | ReplyKeyboardMarkup | ReplyKeyboardRemove | ForceReply | undefined;
         protect_content?: boolean | undefined;
-        allow_sending_without_reply?: boolean | undefined;
```
Comment (XC-Zhang) [inline types/node-telegram-bot-api/index.d.ts:164]: Is this line intentional? Deletion of this line failed the tests (See GitHub pipeline).

**Review by XC-Zhang — CHANGES_REQUESTED** [review]
Either restore Line 164 or change tests.

File: types/node-telegram-bot-api/index.d.ts (hunk 1 above)
Comment (toniop99) [reply types/node-telegram-bot-api/index.d.ts:164]: Is not intentional, will restore it in a moment. Thanks!

**Review by iisaduan — APPROVED** [review]

**Comment by toniop99** [comment]
Ready to merge

**Pull request merged**